compiler/passes/codegen.cpp

```diff
@@ -625,6 +625,20 @@ static void codegen_header_compilation_config() {
     genComment("Filename Lookup Table");
     genFilenameTable();
 
+    //This adds the Chapel symbol table to the file
```

Comment:
As with genFilenameTable, I'd like this code to be in its own function. Also, it'll need to handle LLVM as well, but we can get the C version working first.